spec/javascripts/saveSpec.js

```diff
@@ -255,4 +254,21 @@ describe('Saving a resource instance', function() {
     });
 
   });
+
+  describe('payload for save requests', function() {
+    var resource;
+    beforeEach(function() {
+      resource = Model.create({ name: 'foo', id: 12, subject: 'hello world!' });
+    });
+
+    it('saves requested fields when specified', function() {
+      resource.save({fields: ['name', 'id']});
```

Comment:
Not sure I understand why we'd need to add observers, couldn't we keep a serverCopy whenever we fetched from the server and then compare it with the localCopy when saving? Essentially a smarter version of [EditableResource](https://github.com/zendesk/zendesk_console/blob/master/app/assets/javascripts/lib/editable_resource.module.js). I'm not sure you'd always know what you changed, IIRC consumers share the same instance of each EmberResource model, so different consumers could've changed different properties.